python/cudf/cudf/core/column/column.py

```diff
@@ -963,6 +968,20 @@ def as_numerical_column(
     ) -> "cudf.core.column.NumericalColumn":
         raise NotImplementedError
 
+    def convert_alias(self, dtype):
+        aliases = {
+            "UInt8": "uint8",
+            "UInt16": "uint16",
+            "UInt32": "uint32",
+            "UInt64": "uint64",
+            "Int8": "int8",
+            "Int16": "int16",
+            "Int32": "int32",
+            "Int64": "int64",
+            "boolean": "bool",
+        }
+        return aliases[dtype]
+
```

Comment:
```suggestion
```

Defining this dict inside a function would trigger dict creation everytime we invoke this function. Instead we can add this dictionary mapping in `python/cudf/cudf/utils/dtypes.py` as:
```python
pandas_dtypes_to_cudf_dtypes = {
            "UInt8": "uint8",
            "UInt16": "uint16",
            "UInt32": "uint32",
            "UInt64": "uint64",
            "Int8": "int8",
            "Int16": "int16",
            "Int32": "int32",
            "Int64": "int64",
            "boolean": "bool",
        }
```

We will also have to do special handling for `Float64` & `Float32` here: 
https://github.com/rapidsai/cudf/blob/5ba3ed552a2e499c6a004b483ded5967977dd0d8/python/cudf/cudf/utils/dtypes.py#L95-L99

Reply:
updated - thank you!